src/Cassandra.IntegrationTests/TestClusterManagement/CcmCluster.cs

```diff
@@ -36,6 +36,10 @@ public void Create(int nodeLength, TestClusterOptions options = null)
             }
             _ccm = new CcmBridge(Name, ClusterIpPrefix);
             _ccm.Create(_version, options.UseSsl);
+            if (options.CassandraYaml != null)
+            {
+                _ccm.UpdateConfig(options.CassandraYaml);
```

Comment:
This is not being used, can we skip this change too? I want to avoid unnecessary conflicts with the DSE driver.